### ISSUE TITLE
Invert colors (white-on-black) in either web-server or gotty Terminal.app display for macOS floating browser?

### ISSUE THREAD

**Issue by johnnyutahh** [nicolargo/glances#1518]
How can I get a white background with dark(er) foreground/font for either the web-server or regular display/output? None of the following commands appear to work in my case...

```
$ glances --theme-white -w
$ glances -w --theme-white
$ gotty glances --theme-white
$ gotty glances
```

...because:

I float/"display sticky" the resulting web-server windows using [Helium](https://heliumfloats.com/) or [Flobro](https://flobro.cornips.nl/) so I can keep the `glances` system info "always on top". For some reason, Helium and Flobro make the backgrounds dark, causing my [macOS-inverted-video display](https://support.apple.com/en-us/HT201899) (for late-night/dark-environment viewing) to show the background as white (in both Helium or Flobro), making the glances window rather "blinding" on my eyes, compared to rest of the MacBook display. This is different than [Mojave (macOS version) dark mode](https://support.apple.com/en-us/HT208976).

Maybe this is more an issue with _both_ Helium and Flobro?

 I was not (yet) able to RTFM or github-find any other options.

I've [not yet found a way to invert-colors of only one macOS window/app](https://apple.stackexchange.com/a/32183/13179).

My environment details:

```
$ glances --version
Glances v3.1.1 with psutil v5.6.3
$ sw_vers 
ProductName:	Mac OS X
ProductVersion:	10.14.6
BuildVersion:	18G84
$ brew info glances
glances: stable 3.1.1 (bottled)
Alternative to top/htop
https://nicolargo.github.io/glances/
/usr/local/Cellar/glances/3.1.1 (3,333 files, 56.4MB) *
  Poured from bottle on 2019-07-27 at 14:07:55
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/glances.rb
==> Dependencies
Required: python ✔
==> Analytics
install: 2,525 (30 days), 6,093 (90 days), 21,752 (365 days)
install_on_request: 2,515 (30 days), 6,071 (90 days), 21,701 (365 days)
build_error: 0 (30 days)
$ 
```

**Comment by nicolargo** [comment]
@johnnyutahh  i do not think that the white theme option is available on the Web UI... @notFloran , can you confirm ?

**Comment by notFloran** [comment]
Yes, only the black theme is available on the web interface.
